hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/hip904/TokenClaimAirdropTest.java

```diff
@@ -117,6 +140,98 @@ final Stream<DynamicTest> claimFungibleTokenAirdrop() {
                         getAccountInfo(RECEIVER).hasToken(relationshipWith(NON_FUNGIBLE_TOKEN)));
     }
 
+    @HapiTest
+    @DisplayName("token claim with no pending airdrop should fail")
+    final Stream<DynamicTest> tokenClaimWithNoPendingAirdrop() {
+        return hapiTest(tokenClaimAirdrop().hasPrecheck(EMPTY_PENDING_AIRDROP_ID_LIST));
+    }
+
+    @HapiTest
+    @DisplayName("token claim with duplicate entires should fail")
+    final Stream<DynamicTest> duplicateClaimAirdrop() {
+        return hapiTest(
+                cryptoCreate(OWNER).balance(ONE_HUNDRED_HBARS).maxAutomaticTokenAssociations(0),
+                cryptoCreate(OWNER_2).balance(ONE_HUNDRED_HBARS).maxAutomaticTokenAssociations(0),
+                cryptoCreate(RECEIVER).balance(ONE_HUNDRED_HBARS),
+                tokenCreate(FUNGIBLE_TOKEN)
+                        .treasury(OWNER)
+                        .tokenType(FUNGIBLE_COMMON)
+                        .initialSupply(1000L),
```

Comment:
OWNER and FUNGIBLE_TOKEN are already created in `beforeAll()`. You can use them directly.